public/doc/swagger-2.yaml

```diff
@@ -480,9 +509,9 @@ paths:
         '401':
           $ref: '#/responses/Unauthorized'
         '403':
-          $ref: '#/responses/Forbidden'  
+          $ref: '#/responses/Forbidden'
         '404':
-          description: order not found    
+          description: order not found
```

Comment:
Very small nit.  Can we add capitalization to the first word? `/order/Order/` that will keep our documentation consistent.

Reply:
@mkanoor - Looks like there is 1 more on line 514.